external/sketch/ansible/roles/proxypass-setup/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+# tasks file for proxypass-setup
+- name: Copy ProxyProtocol to Sketch Nodes
+  copy:
+    src: files/install_ProxyProtocol.sh
+    dest: "{{ ansible_user_dir }}/install_ProxyProtocol.sh"
+    mode: 0755
+
+- name: Run ProxyProtocol on Edge nodes
+  become: yes
+  shell: "./install_ProxyProtocol.sh {{ hostvars[groups['middle'][0]]['middle_host_ip'] }} edge"
```

Comment:
Only one middle will ever be used here, is that intended? What if there are multiple middles?